rs3_trading/database/ge_tick_database.py

```diff
@@ -10,8 +10,16 @@ def create_volume_table(con: DuckDBPyConnection):
     con.execute('create table if not exists rs3_volume_data(datetime_utc datetime, item varchar, item_name varchar, volume long, primary key(item, datetime_utc))')
 
 
-def insert_into_table(con: DuckDBPyConnection, table_name: str, column_names: str, dataframe: pd.DataFrame):
+def create_historical_data_table(con: DuckDBPyConnection):
+    con.execute('create table if not exists rs3_historical_data(id int, datetime_utc datetime, price long, volume long, item_name varchar)')
+
+
+def create_id_item_name_table(con: DuckDBPyConnection):
+    con.execute('create table if not exists rs3_ids_and_item_names(id int, item_name varchar)')
```

Comment:
Id make a function that calls your all your make table functions at once. So you can just call like init_tables and it will do all of them

Reply:
I have that already. The function is in the utils